### ISSUE TITLE
Hedges added by Every Compat have missing item textures

### ISSUE THREAD

**Issue by NJL5390** [MehVahdJukaar/WoodGood#5]
Every Compat Version: 1.3.2
Selene Version: 1.17.4
Latest Log: https://gist.github.com/NJL5390/b8c45d463a6b89b047679502d8268a81

Only noticed this after updating Every Compat to the latest version. Just the item textures are missing, the block textures are totally fine, it's the textures for the items that seem to be missing.
![2022-06-09_17 45 00](https://user-images.githubusercontent.com/64381954/172903553-bb8f0c94-145c-4461-9f8e-3868d38fd9aa.png)
![2022-06-09_17 45 10](https://user-images.githubusercontent.com/64381954/172903570-e65510dc-96fd-4a72-a45e-2f8eb986c662.png)
![2022-06-09_17 55 10](https://user-images.githubusercontent.com/64381954/172903588-45d7a5ec-1f93-4db9-b996-24eda174b4e8.png)
This only started happening when I updated the mod to the latest version, everything was working fine beforehand.

Thanks for the fantastic mod! I've been needing a mod like this for a very long time so thank you very much for making it!

**Comment by MehVahdJukaar** [comment]
that's a weird issue. I've seen somebody else have it but I couldn not replicate it. Can you send me your log?

**Comment by NJL5390** [comment]
Here's my latest log: https://gist.github.com/NJL5390/b8c45d463a6b89b047679502d8268a81

I reset the every compat config file before I updated in case that has anything to do with it.

**Comment by MehVahdJukaar** [comment]
oh sorry I did't see you already sent one. Unfortunately it doesnt show much. What mods do you have that add wood types? It might be that when one of them fails all other models fail to get added too

**Comment by NJL5390** [comment]
For the session that generated the log that I sent, I just used Blue Skies, Quark and Twilight Forest. 

So it could be one of those, or all of them. I haven't had a chance to test much. I'll be sure to update you if I do get around to doing more testing though!

**Comment by MehVahdJukaar** [comment]
Check the newest version. Might have been fixed

**Comment by NJL5390** [comment]
I'm afraid not. 
![2022-06-10_15 29 59](https://user-images.githubusercontent.com/64381954/173089370-fc0833c0-8d99-423b-b287-a353949be8fd.png)

**Comment by MehVahdJukaar** [comment]
hm damn I was sure I did fix a similar issue. Can you try without blue skies and maybe try adding other mods that add wood types too?
Last issue I fixed happened when one had too little wood types oddly enough

**Comment by MehVahdJukaar** [comment]
I released yet another update and I still cant personally replicate this. Can you confirm if it's still there?

**Comment by NJL5390** [comment]
Unfortunately, It still is.
![2022-06-12_00 13 34](https://user-images.githubusercontent.com/64381954/173207880-4624289b-f8c1-4a40-b8c6-6d4238b2208c.png)



> hm damn I was sure I did fix a similar issue. Can you try without blue skies and maybe try adding other mods that add wood types too? Last issue I fixed happened when one had too little wood types oddly enough

In the screenshot you can see that I added several biome mods to up the wood type amount and that did not seem to affect the issue at all. 

I also tested without Blue Skies earlier and that didn't affect the issue at all. 

Sorry for the long delay between replies, I'm fairly busy with IRL stuff at the moment, so I'm just updating you on this whenever I can.

**Comment by BookerCatch** [comment]
Every Compat Version: 1.5.0
Selene Version: 1.17.6
Latest Log: https://gist.github.com/BookerCatch/dce8e640cafcffdb7e17d17d814e528a#file-gistfile1-txt-L386

I'm getting the same issue. Worth checking https://github.com/MehVahdJukaar/WoodGood/blob/fe9b82678eee664f2720c199c3b5d23eb67c5afe/src/main/java/net/mehvahdjukaar/every_compat/modules/valhelsia_structures/AxeCraftingRecipeTemplate.java#L58

And consequently line 166 [here](https://github.com/MehVahdJukaar/Moonlight/blob/2b39631173df6dc17c881320b956dbe132fcb688/src/main/java/net/mehvahdjukaar/selene/block_set/BlockType.java#L166).

**Comment by MehVahdJukaar** [comment]
I'm aware of where its failing, I just dont know how as it seems it would be impossible to me. I added more debugging logs. I cant fix this until I'm able to personally replicate it (I still cant, both the recipe and texture work for me)

**Comment by MehVahdJukaar** [comment]
Ok it should be fixed. Still I have little idea of why that happening. Oak hedge item was somehow getting nuked without throwing errors and at random and I still don't know why

**Comment by BookerCatch** [comment]
Biome Makeover 1.18.2-1.4.24
Every Compat 1.18.2-1.5.1
Quark 3.2-357
Selene 1.18.2-1.17.7

https://gist.github.com/BookerCatch/036f742b648598206537e5618184148f

![image](https://user-images.githubusercontent.com/43504436/173275614-8dffbdb6-b77e-447f-ad90-7e19c2545c24.png)

I'm afraid the issue is still present.

**Comment by MehVahdJukaar** [comment]
I literally have no idea anymore then. Do oak hedges show up?

**Comment by MehVahdJukaar** [comment]
Ok I believe I actually fixed this

**Comment by BookerCatch** [comment]
I was preparing a message just now. I was getting the exact same results. Oak hedges don't show up in the creative menu, but are accessible via the /give command.

I cloned the repo to reproduce the bug, but everything worked as expected. I then created a client instance with the exact mod loadout used in the dev environment and everything worked as expected.

I then removed Macaw's mods and the issue reocurred once again. I'm adding them back one by one to find the culprit.
